### ISSUE TITLE
Changes in Views inside Practice Tab [A11y]

### ISSUE THREAD

**Issue by Arjupta** [oppia/oppia-android#2326]
**Describe the bug**

Views inside the Practice Tab are not fully compatible with some users from the Accessibility point of view.
![Screenshot_2020-12-17-11-27-30-51_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/103155329-e705a100-47c4-11eb-9af2-2bc829d22c79.jpg)
Issue 1 - The contrast ratio of the topic practice start button is not correct in the inactive state.
Issue 2 - The subtopic checkbox has a height of 24dp which is less than the standard one (48dp)

**To Reproduce**
Steps to reproduce the behaviour:
1. Start the Accessibility Scanner App on your device
2. Open the Oppia Android App and start Fraction Exploration
3. Switch to the Practice Tab
3. Scan the screenshots with the Accessibility Scanner
4. You will see the tests that are not passed by that screen

**Expected behaviour**
All the tests must be passed by scanning the screenshots for both Tablet and Mobile Devices in every orientation.

**Screenshots**
|Issue 1|Issue 2|
|:-:|:-:|
|![Screenshot_2020-12-17-11-27-02-40_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/103155332-e836ce00-47c4-11eb-9150-f18329072e55.jpg)|![Screenshot_2020-12-17-11-26-53-15_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/103155360-31871d80-47c5-11eb-89c9-6647beae3a6d.jpg)

**Device**
- Real Device - RealMe3
- SDK version - 29

**Additional context**
The Screenshots are descriptive enough for understanding the issues but still, if there is a need of clarification then you can comment here. Multiple Pull Requests can also be made for solving these issues.

Certain Views does not have the correct contrast ratios (Inactive state) as mentioned in the screenshots. To solve this, one must find a way to make the views disabled instead of being in non-clickable state only

Reference - [https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner](https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner)

**Comment by Mehul-Bisht** [comment]
I was able to fix the issue with the checkboxes, Can I be assigned for this issue?

**Comment by FareesHussain** [comment]
@ARJUPTA is this ok to work on this
I think accessibility issues are blocked and the PR related are not to be merged now

**Comment by Mehul-Bisht** [comment]
Please update the status to `BLOCKED` to clarify which issues are not open currently

**Comment by Mehul-Bisht** [comment]
@ARJUPTA @FareesHussain should I not be working on this issue anymore then? (and also others which mention [A11y] in their title)

**Comment by FareesHussain** [comment]
@rt4914 are accessibility issues open to work now?

**Comment by Arjupta** [comment]
@Mehul-Bisht @FareesHussain this particular issue can be solved with the given description. This is why it is not blocked on anything. @Mehul-Bisht you can comment here if you still want to work on this.

**Comment by Mehul-Bisht** [comment]
Yes as I mentioned earlier, I was able to fix the issue with checkboxes for portrait mode but was waiting for the confirmation, so I did not fix the issue for other orientations yet. I'll do that now.

**Comment by Arjupta** [comment]
### Checklist
 (Contributors can take one issue at a time)
* [ ]  `Issue-1` (not assigned to anyone)
* [x]  `Issue-2` (already done)

**Comment by Arjupta** [comment]
@Mehul-Bisht as you are now able to run Tablet Emulators , you can test and create a new PR for this part of the issue you were assigned to https://github.com/oppia/oppia-android/pull/2525#issuecomment-766391600

**Comment by Mehul-Bisht** [comment]
> @Mehul-Bisht as you are now able to run Tablet Emulators , you can test and create a new PR for this part of the issue you were assigned to [#2525 (comment)](https://github.com/oppia/oppia-android/pull/2525#issuecomment-766391600)

Does it require a separate branch to be created or I can make a PR to the same branch also?

**Comment by ARJUPTA** [comment]
> > @Mehul-Bisht as you are now able to run Tablet Emulators , you can test and create a new PR for this part of the issue you were assigned to [#2525 (comment)](https://github.com/oppia/oppia-android/pull/2525#issuecomment-766391600)
> 
> Does it require a separate branch to be created or I can make a PR to the same branch also?

Make a new branch, it won't actually make a difference.

**Comment by Arjupta** [comment]
> > @Mehul-Bisht as you are now able to run Tablet Emulators , you can test and create a new PR for this part of the issue you were assigned to [#2525 (comment)](https://github.com/oppia/oppia-android/pull/2525#issuecomment-766391600)
> 
> 
> Does it require a separate branch to be created or I can make a PR to the same branch also?
> 
 Make a new branch, it won't actually make a difference as your code is already merged

**Comment by Arjupta** [comment]
@Mehul-Bisht any update?

**Comment by Mehul-Bisht** [comment]
> @Mehul-Bisht any update?

@ARJUPTA I have made a PR for tablet layouts here: #2797
Kindly confirm with Rajat once and notify me about any required changes.
